docs/operations/migrating/docker/postgresql-migration.md

```diff
@@ -1,6 +1,6 @@
 # Migrating your Docker OpenProject database to PostgreSQL
 
-This guide will migrate your docker-based MySQL installation to a PostgreSQL installation using [pgloader](https://github.com/dimitri/pgloader). 
```

Comment:
Please move this somewhere to https://github.com/opf/openproject/tree/dev/help/installation-and-operations so the new help picks it up. The docs folder will be removed

Reply:
Should be put within https://github.com/opf/openproject/tree/dev/help/installation-and-operations/misc